packaging/debian/datadog-agent.init

```diff
@@ -151,6 +152,11 @@ case "$1" in
         exit $?
         ;;
 
+    reload)
+        kill -12 `cat $COLLECTOR_PIDFILE`
```

Comment:
as leo pointed out using SIGHUP has actually became a de facto standard, we should use it.